content/working_with/console/widgets/sitesMap.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bt_wiki
+title: Sites Map
+category: Cloudify Console
+draft: false
+---
+Displays the world map with defined sites marked. Only sites with defined location are displayed on the map.
+
+![sitesMap]( /images/ui/widgets/sitesMap.png )
+
+Color of the site marker on the map indicates site health. It is derived from status of deployments attached to the site.
+ 
+After clicking on site marker on the map, user can see popup with site details.
+
+![sitesMap]( /images/ui/widgets/sitesMap_siteDetails.png )
+
+Site details contain information about deployments' statuses, indicated as follows:
+
+* **Green** - number of deployments with all nodes successfully started,
+* **Yellow** - number of deployments in which active workflow execution is performed,
+* **Blue** - number of deployments with non-started nodes,
+* **Red** - number of deployments with failed workflow execution. 
+
+{{% tip title="Tip" %}}
+If you have problems displaying map, maybe you need to change map provider. Check [this page]({{< relref "working_with/console/advanced-configuration.md" >}}) for more information.
+{{% /tip %}}  
```

Comment:
I think we should remove this section. I refer to the option to edit the map provider as a "support" feature rather than a customer feature.
Also, I could not find the document you refer to, (which we should probably hide as well).

Reply:
The description about configuration is not merged, yet.
It was added in separate PR: https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1175

But OK, I removed the tip from the widget's description.